data/mods/Xedra_Evolved/mutations/vampire_trait_eocs.json

```
@@ -46,6 +46,13 @@
         "targeted": true
       }
     ]
+  },
+    {
+    "type": "effect_on_condition",
+    "id": "EOC_COMMUNE_NIGHT_MAP",
+    "condition": { "and": { "compare_int": [ { "u_val": "pos_z" }, ">=", { "const": 0 } ] },  "is_night": true, "u_is_outside" },
```

```suggestion
    "condition": { "and": [ { "not": "is_day" }, "u_is_outside" ] },
```
You shouldn't need the Z-level check if you already have is_outside, I don't think. I went and tested it and with the Z level check you can't use this if you're standing in a crater.